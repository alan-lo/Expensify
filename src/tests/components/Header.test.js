import React from 'react';
// import ReactShallowRenderer from 'react-test-renderer/shallow';
import { shallow} from 'enzyme';
import Header from '../../components/Header';

test('should render header correctly', ()=>{
  const wrapper = shallow(<Header/>);
  expect(wrapper).toMatchSnapshot();
  // const renderer =new ReactShallowRenderer();
  // renderer.render(<Headergit./>);
  // expect(renderer.getRenderOutput()).toMatchSnapshot();
})