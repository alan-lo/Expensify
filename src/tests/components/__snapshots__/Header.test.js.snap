// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header correctly 1`] = `
<div>
  <h1>
    Expensify
  </h1>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    exact={true}
    to="/"
  >
    Go Home
  </NavLink>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    to="/create"
  >
    Create Page
  </NavLink>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    to="/help"
  >
    Help Page
  </NavLink>
</div>
`;
